_posts/2019-02-11-bootstrap-4-3-0.md

```diff
@@ -0,0 +1,60 @@
+
+[Head to to the v4.3.x docs](https://getbootstrap.com/docs/4.3/) to see the latest in action. The full release has been published to npm and will soon appear on the Bootstrap CDN and Rubygems.
+
+## Introducing responsive font sizes
+
+![Responsive font-sizes](https://camo.githubusercontent.com/eb6458d9bb840c5b5764a33ac626ca084f12e460/68747470733a2f2f692e696d6775722e636f6d2f50326978614b482e706e67)
+
+Our biggest new addition to Bootstrap in v4.3 is responsive font sizes, a [new project in the Bootstrap GitHub org](https://github.com/twbs/rfs) to automate calculate an appropriate `font-size` based on the dimensions of a visitor's device or browser viewport. Here's how it works:
+
+- All `font-size` properties have been switched to the `@include font-size()` mixin. Our Stylelint configuration now prevents the usage of `font-size` property.
+
+- Disabled by default, you can opt into this new behavior by toggling the `$enable-responsive-font-sizes` boolean variable.
+
+- The base `font-size` is set via the `$rfs-base-font-size` Sass variable. Modifying that value and recompiling will automatically modify any element's font-size when using the new mixin.
+
+We've enabled responsive font-sizes in our own docs starting with v4.3. Please share feedback with us via GItHub issues or on Twitter. We've added some light guidance to [our Typography docs](https://getbootstrap.com/docs/4.3/content/typography/#responsive-font-sizes) to explain the feature. You can also learn more by [reading the rfs project documentation](https://github.com/twbs/rfs).
+
+## Branches, Hugo, and jQuery
+
+Right after shipping v4.3, we'll be tackling a few key changes on our road to active v5 development. These are larger changes to how we maintain and develop Bootstrap and are considered foundational for v5.
+
+- **Improving our branches for development.** `master` will become our new `v3-dev` branch (used only for critical security fixes). `v4-dev` will stay as-is, but we'll cut a new `master` branch from there to develop v5.
+
+- **We've moving to Hugo!** Jekyll has been great, but it's starting to slow us down in local development. We'll be making changes to our dependencies to support this move, and there's already a pull request in progress and near completion for the change. [Follow along](https://github.com/twbs/bootstrap/pull/28014) to see what's changing.
+
+- **We're dropping jQuery for regular JavaScript.** The cat is out of the bag—we're dropping our largest client-side dependency for regular JavaScript. Similar to the Hugo move, we've been working on this for a long time and have a [pull request in progress](https://github.com/twbs/bootstrap/pull/23586) and near completion.
+
+We'll have even more to share soon around v5's plans after we tackle these bigger items. In the meantime, keep the feedback coming on GitHub and Twitter!
+
+<3,<br>
+[@mdo](https://twitter.com/mdo) & [team](https://github.com/twbs)
```

Comment:
Maybe we should link to the people page here?

```suggestion
[@mdo](https://twitter.com/mdo) & [team](https://github.com/twbs/people)
```

Reply:
If we go with this, we should update all other posts too later.

Reply:
Alternatively, we can go with `https://getbootstrap.com/about/` which will redirect to the latest docs version about page after 4.3.

Reply:
Let's leave for now and tackle in follow-up so that it's uniformly applied. We can probably put this in the post template tbh. Opened #107 to track.